bin/newver

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -eu
+
+if [ $# -eq 0 ]; then
+  >&2 echo "Usage: 'bin/newver X.Y.Z', where X.Y.Z is the Rubocop version to add."
+  exit 1
+fi
+NEWVER="$1"
+
+git checkout Gemfile*
+git checkout -b "newver/${NEWVER}"
+
+ed Gemfile <<-EDITS
+/gem "parser",/c
+gem "parser"
+.
+/gem "rubocop",/c
+gem "rubocop", "~> ${NEWVER}", require: false
+.
+wq
+EDITS
+
+bundle update
```

Comment:
wdyt about doing this in docker? In general we try to avoid needing to have non-docker things installed on your host. We could do something like:

```shell
docker run --user root --rm --volume "$(pwd)":/usr/src/app --workdir /usr/src/app codeclimate/codeclimate-rubocop bundle update rubocop
```

In a lot of our code bases, we've added such a task to the Makefile as `make bundle`. If we did that, we might invoke it here as `make bundle -e BUNDLE_ARGS='update rubocop'`

Reply:
That seems like a nice improvement; I will change it to invoke docker like your example.